frontend/src/redux/selectors/resource.js

```diff
@@ -16,7 +16,19 @@ const resourceDescription = (resource) =>
     ? resource.skills
     : resource.description) ?? "";
 
-const strCompare = (a, b) => a.localeCompare(b);
+const strCompare = (a, b) => {
+  if (a === "" || a === null) {
+    return 1;
+  }
+  if (b === "" || b === null) {
+    return -1;
+  }
+  if (a === b) {
+    return 0;
```

Comment:
huh. never knew that `localeCompare` didn't include this falsy / equality checks!

Reply:
Glad you brought this up! :D 
It actually does by default, however it sorts the empty string such that it appears first in a list, so `["C","D","A","B",""]` sorts to `["","A","B","C","D"]` by default, which means all the empty string entries appear first. In an app like this where we could have hundreds of blank entries for a column, it makes sense to put the empty string last.